README.md

```diff
@@ -1,4 +1,6 @@
-# data_example_microscopy_sem
-Example dataset containing scanning electron microscopy (SEM) data to illustrate BIDS convention.
+# data_axondeepseg_sem
+AxonDeepSeg scanning electron microscopy (SEM) dataset
+10 rat spinal cord samples with axon and myelin manual segmentation labels.
```

Comment:
it would be great to have more information about the origin of these scans: are they from us? if so, can we maybe add a link to duke so we don't duplicate these images in the future if we are planning on adding more data from duke in the future for enriching our database

Reply:
This is a very good point with a complicated answer.

The best "provenance" I could find is on `duke/projects/axondeepseg/raw_data/data_SEM/osf`. This is not very helpful because it is the exact same thing as what is currently on OSF as our SEM dataset with no additional info. It also matches only partially with the image descriptions in the ADS paper, some images match, others don't. So the dataset somehow changed between the paper and today.

For this dataset, the names `rat1` to `rat8` are arbitrary and were added by us when creating the example dataset. So the only "real" identification we have is the sample name (ex: `data14`). Only `rat3` has 3 samples because it matched the paper as being all from the same rat (we've discussed this before, I did not find where though).

What does `data14` means? Answer: I had no idea. So I looked in the google drive `AxonDeepSeg/data/data_sem/with_Groundtruth` and for some images (but not all) there is a sneaky `info` file which seems to point to the original file name (ex for `data5` --> `20160628_rat_28`).

Where are those files? Probably in `duke/projects/axondeepseg` itself or in `duke/histology/rat`. This folder contains hundreds of big panorama SEM images split in mosaic of tens of chunks.

Short answer: I'll do my best to find the provenance and add information. We may want to revisit the `sub-id` afterwards.

As for enriching our database, it seems that we have plenty to choose from, from a brief overview, most of those mosaic have been segmented with `AxonSeg` before.

Reply:
awesome, thank you very much for looking into this @mariehbourget . If you need help @alexfoias can you please chip in

Reply:
I have done a first pass, going through the `ADS google drive`, `duke/histology` and `duke/projects/axondeepseg`.
Here is a breakdown of my findings: https://docs.google.com/spreadsheets/d/1nc_H3TecCreJ4RSRYOOjiFvXD3Q5jfWsLHdutLDxpTk/edit#gid=0

- 2 images are found.
- 4 images are from a panorama, we would have to go through the mosaic of this panorama to find the exact source image for each.
- 2 images are not found, most likely part of the many mosaics available in `duke/histology/rat`.
- 2 images are found with discrepancies in naming between the dataset and the source image.

Reply:
I would suggest we move on and release this version + open an issue to follow-up with the provenance.